modules/auth/proc-authenticationg-with-the-guest-user.adoc

```diff
@@ -0,0 +1,15 @@
+[id="authenticating-with-the-guest-user"]
+= Authenticating with the Guest user
+
+To start exploring {product} features, do not worry about authentication: after the default installation, you can authenticate with the Guest user, and use all the application features.
```

Comment:
So a user doesn't need to authenticate to explore {product}, but the Prereq require installation. So is this procedure to help them explore the product, or to perform a default installation, and be able to log in as a Guest?

Reply:
```suggestion
To start exploring {product} features, do not worry about setting up authentication: after the default installation, you can authenticate with the Guest user, and use all the application features.
```

Reply:
As a user, why do I want to follow the rest of this procedure if I'm just exploring? I don't quite get the purpose of "To start exploring {product} features, do not worry about setting up authentication" as an intro. It seems unnecessary.

Or do you mean a user doesn't have to setup their company network authentication with RHDH, but instead they can they can create a Guest user, and use Guest to poke around to see if RHDH provides a solution for their need?